src/bundle/Resources/views/user/settings/update_datetime_format.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends '@ezdesign/user/settings/update.html.twig' %}
+
+{% trans_default_domain 'user_settings' %}
+
+{% block form %}
+    {{ parent() }}
+
+    <div id="ez-datetime-format-preview" class="alert ez-alert--info mt-4" role="alert">
+        {{ 'settings.datetime_format.preview_label'|trans|desc('Date & Time will be displayed in this format:') }}
+        <span class="ez-datetime-format-preview-value" />
+    </div>
+{% endblock %}
+
+{% block javascripts %}
+    {{ encore_entry_script_tags('ezplatform-admin-ui-settings-datetime-format-update', null, 'ezplatform') }}
```

Comment:
Maybe it would be ok to use here `parent()` as well? Or we are sure we don't want the JS from `update.html.twig`?